notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -12041,6 +12041,51 @@ fear [There were pointed moment when Jillian was in terror of her husband, Spenc
 incorporeal being [Spencer's body had been taken over by a malevolent alien entity that, at least according to Sherman Reese, was traveling space as a transmission at the speed of light.]
 
 
+movie: Peutetre (1999)
+======================
+
+:: Title
+Peut-être
+
+:: Description
+Peut-être (Maybe; Perhaps) is a 1999 French science fiction comedy film. After
+having sex with his girlfriend Lucie in a bathroom, Arthur discovers that a
+ceiling panel is a time portal to the Paris in the future, although it appears
+more like a sun-baked desert city by that point. There he meets an old man
+named Ako who turns out to be Arthur's son. Ako attempts to persuade Arthur to
+impregnate Lucie so that he can exist in this future.
+
+:: Date
+1999-11-10
+
+:: References
+https://en.wikipedia.org/wiki/Peut-%C3%AAtre
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+what if I met my distance descendants [The main novelty of the film concerns the young man Arthur meeting his future children and grandchildren after passing through a time portal.]
+father and son [The young man Arthur traveled into the future whereupon he met his future son, Ako, who was by then an old man. The story then follows Ako's attempts to persuade Arthur to impregnate his girlfriend so that Ako and his progeny can continue to exist in the future.]
+
+:: Major Themes
+what if I found myself in the future [Arthur passed through a time portal to find himself 60 years in the future. Not only did he find the Paris of the future to be a sun-baked desert, but he also met his future son and grandchildren.]
+personal practical dilemma [Should Arthur impregnate his girlfriend or not? If he did, then his future son Ako and Ako's descendants would go on living happily in the future; otherwise they would cease to exist.]
+facing death [Ako was faced with the prospect of vanishing into thin air should he fail in his attempts to convince his son, Arthur, to impregnate his mother, Lucie.]
+boyfriend and girlfriend [Arthur and his girlfriend Lucie found themselves in a most strange predicament while attending a wild, all-night millennium party.]
+
+:: Minor Themes
+time portal [Arthur discovered a bathroom ceiling panel was actually a time portal and subsequently passed through it to find himself about 60 years in the future.]
+husband and wife [Ako introduced Arthur to his wife Marie-Jeanne.]
+father and daughter [Ako introduced Arthur to Arthur's future daughter. Rosemonde tried to keep her crazy, gun-toting father under control.]
+brother and sister [Ako briefly quarreled with his sister Madeline.]
+illegal drug trade [The millennium party goes were shown smoking and snorting various street drugs.]
```

Comment:
Ahem, that's not a motivation for the crime of illegal drug trade. If we saw drugs clandestinely (and pointedly so) change hands in return for money, then okay. 

Maybe:

psychoactive drug experience
drug abuse

Were topical?

Reply:
I've knowingly been using **illegal drug trade** to catch these situations with the understanding (for me) that stock will be taken later to bud out a new and more appropriate theme. So I suggest it be left there for the moment, and that I create a list to be reviewed.

Reply:
The missing drug theme. I suppose, would be **recreational drug use** -> **recreational activity**. We could introduce that one here, and I could move such instances out from **illegal drug trade**.

Reply:
I'm washing my hands of this funny-business.

Reply:
I think most people would be satisfied with the "drug abuse" theme to cover the illicit use of such controlled substances. I realise I'm talking to a pro-drug hippie who feels the word "abuse" is amiss, however, as the detrimental effects are not obvious in the stories under scrutiny. I don't have any objection to a recreational drug use theme. How it relates to smoking and drinking, is worth taking some care with.

Reply:
I proposed the new theme **recreational illicit drug use** in this branch.

Where I'm coming from is that I don't want to imagine what most people think, but rather what is going on in stories that is worth capturing. And there are stories where people, say, smoke pot (at least when that used to be illegal) in the same way that people have a beer after work. We don't really have a theme to cover that.